pages/common/vegeta.md

```diff
@@ -0,0 +1,20 @@
+# vegeta
+
+> HTTP load testing tool.
+> Vegeta is a versatile HTTP load testing tool built out of a need to drill HTTP services with a constant request rate. It can be used both as a command line utility and a library.
```

Comment:
Perhaps we can make this more concise? We've mentioned that it's a load-testing tool in the first line of the description.

Reply:
@antoine-amara @sbrl how about:
```suggestion
> Vegeta is both a command line utility and a library for HTTP load testing.
> See also `ab`.
```

Reply:
Sounds good to me! Maybe cut the `Vegeta is` from the beginning though, as the program name is literally the page title :P